cli/commands/config.go

```diff
@@ -2,6 +2,7 @@ package commands
 
 import (
 	"fmt"
+
```

Comment:
My editor uses `goimports` (which is identical to `gofmt` except it organises imports too). I quite like it (it separates third party libraries from official libraries and removes unused imports) - but let me know if you'd like me  to stick with just `gofmt`.